Slim/Control/Queries.pm

```diff
@@ -4697,6 +4697,11 @@ sub worksQuery {
 		}
 	}
 
+	if ( defined $workID ) {
+		push @{$w}, "works.id = ?";
+		push @{$p}, $workID;
+	}
```

Comment:
Looking into @frank1969b's issues I was wondering whether filter by `tracks.work` wouldn't be more efficient?